### PR TITLE
py3 div: some of the trickier changes

### PR DESCRIPTION
**Pull request by nickpell**
@dimagi/py3

### CODE REVIEW

File: custom/bihar/reports/display.py [hunk 1]
```diff
@@ -632,7 +633,7 @@ def dob_age(self):
                     dob = dob["#value"]
                 days = (date.today() - CaseDisplay.parse_date(self, dob).date()).days
                 dob = self.parse_date(dob)
-                return "%s, %s" % (dob, int(days/365.25))
+                return "%s, %s" % (dob, int(days / 365.25))
```
Comment (emord) [inline custom/bihar/reports/display.py]: was this supposed to be `//`?

File: corehq/apps/reports/standard/monitoring.py [hunk 2]
```diff
@@ -1629,11 +1630,11 @@ def _rows_by_group(self, report_data):
                     sum(
                         [int(report_data.avg_submissions_by_user.get(user["user_id"], 0))
                         for user in users]
-                    ) / self.num_avg_intervals
+                    ) // self.num_avg_intervals
                 ),
                 # Active users
                 util.numcell("%s / %s" % (active_users, total_users),
-                             value=int((float(active_users) / total_users) * 10000) if total_users else -1,
+                             value=int((float(active_users) // total_users) * 10000) if total_users else -1,
```
Comment (millerdev) [inline corehq/apps/reports/standard/monitoring.py]: I think this one should not be using integer division (it wasn't before).

File: custom/bihar/reports/display.py (hunk 1 above)
Comment (nickpell) [reply custom/bihar/reports/display.py]: I think ```int(days / 365.25)``` and ```int(days // 365.25)``` should always return the same value

File: corehq/apps/reports/standard/monitoring.py (hunk 2 above)
Comment (nickpell) [reply corehq/apps/reports/standard/monitoring.py]: Good catch: 4c0cf5e

**Comment by nickpell** [comment]
✅

**Pull request merged**